ui/app/mainui/activitycenter/popups/ReviewContactRequestPopup.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Utils 0.1 as CoreUtils
+
+import shared 1.0
+import utils 1.0
+
+StatusModal {
+    id: root
+
+    property var notification
+
+    signal accepted
+    signal declined
+
+    anchors.centerIn: parent
+    width: 638
+    padding: Style.current.bigPadding
+
+
+    property StatusMessageDetails messageDetails: StatusMessageDetails {
```

Comment:
Is it necessary to construct this component here? Do we reuse those properties a lot or we could just set the values directly in the right places below?